src/solver/sLR.ml

```diff
@@ -401,11 +401,11 @@ module Make0 =
           let use_box = (not (V.ver>1)) || HM.mem wpoint x in
           let restart_mode_x = h_find_default restart_mode x (2*GobConfig.get_int "solvers.slr4.restart_count") in
           let rstrt = use_box && (V.ver>3) && D.leq tmp old && restart_mode_x <> 0 in
-          if tracing then trace "sol" "Var: %a\n" S.Var.pretty_trace x ;
-          if tracing then trace "sol" "Contrib:%a\n" S.Dom.pretty tmp;
+          if tracing then trace "sol" "Var: %a" S.Var.pretty_trace x ;
+          if tracing then trace "sol" "Contrib:%a" S.Dom.pretty tmp;
           let tmp = if use_box then box old tmp else tmp in
           if not (D.eq tmp old) then begin
-            if tracing then trace "sol" "New Value:%a\n\n" S.Dom.pretty tmp;
+            if tracing then trace "sol" "New Value:%a" S.Dom.pretty tmp;
```

Comment:
Double newline probably on purpose to separate potentially long printout of value from rest?